src/plugins/files/server/usage/integration_tests/usage.test.ts

```diff
@@ -46,26 +46,6 @@ describe('Files usage telemetry', () => {
 
     const { body } = await request.get(root, `/api/stats?extended=true&legacy=true`);
 
-    expect(body.usage.files).toMatchInlineSnapshot(`
```

Comment:
I believe this is testing the collection algorithm, do you mind changing the request to `POST /api/telemetry/v2/clusters/_stats`, with the body `{ unencrypted: true }`?

Then, the same output should be found in `body.stack_stats.kibana.plugins.files`

Reply:
I updated this test.
I still have few of the FTRs failing, trying to read non existing properties, so thinking if the best approach is to remove them

Reply:
Those FTRs need to migrate to a different helper to fetch the telemetry (sorry, I didn't anticipate so many side-effects in FTRs when I added the `good first issue` label). 

I migrated those tests to the new helper in #151808. Once merged, you may rebase your PR and it should get a green CI :)